sites/fast-website/src/app/components/fast-frame/fast-frame.template.ts

```diff
@@ -62,6 +60,7 @@ export const FastFrameTemplate = html<FastFrame>`
                                     html<string>`
                                         <site-color-swatch
                                             tabindex="0"
+                                            aria-label="background color"
```

Comment:
Should these read the hexadecimal color as part of the aria-label so that information is accessible?

Reply:
yep, thanks for bringing that up @nicholasrice . I should have made a note that it was an option I also had. Will make the change.